cloudscheduler/openstackcluster.py

```diff
@@ -238,14 +241,52 @@ def vm_create(self, vm_name, vm_type, vm_user, vm_networkassoc,
                     netid = []
         else:
             netid = []
-        # Need to get the rotating hostname from the google code to use for here.  
+        # Need to get the rotating hostname from the google code to use for here.
         name = self._generate_next_name()
         instance = None
 
         if name:
+            log.info("Trying to create VM on %s: " % self.name)
             try:
-                instance = nova.servers.create(name=name, image=imageobj, flavor=flavor, key_name=key_name, 
-                                               availability_zone=self.placement_zone, nics =netid, userdata=user_data, security_groups=sec_group)
+                if not boot_volume:
+                    instance = nova.servers.create(name=name,
+                                                   image=imageobj,
+                                                   flavor=flavor,
+                                                   key_name=key_name,
+                                                   availability_zone=self.placement_zone,
+                                                   nics =netid,
+                                                   userdata=user_data,
+                                                   security_groups=sec_group)
+                else:
+                    bdm = None
+                    log.debug("creating boot volume")
+                    try:
```

Comment:
Memo for a similar implementation in v2:
during my testing I got the impression that nested try except won't work, this should be checked and reviewed before we move this into V2

Reply:
Yeah I had trouble with nested try except blocks in the past. Let me try to find the type of exceptions cinder throws ... maybe can be smarter here.